pages/docs/integrations/litellm/tracing.mdx

```diff
@@ -80,16 +80,13 @@ You can add additional Langfuse attributes to the requests in order to group req
 Instead of the proxy, you can also use the native LiteLLM Python client. You can find more in-depth documentation in the [LiteLLM docs](https://litellm.vercel.app/docs/observability/langfuse_integration).
 
 <Callout type="warning">
-  LiteLLM relies on the <strong>Langfuse Python SDK v2</strong>. It is currently
-  <strong>not compatible</strong> with the newer
-  <strong>[Python SDK v3](/docs/sdk/python/sdk-v3)</strong>. Please refer to the
-  [v2 documentation](/docs/sdk/python/low-level-sdk) and pin the SDK version
-  during installation:
+  LiteLLM `langfuse` integration relies on the <strong>Langfuse Python SDK v2</strong>. To use 
+  litellm with the newer
+  <strong>[Python SDK v3](/docs/sdk/python/sdk-v3)</strong>, use `langfuse_otel` integration instead.
+
+  You need at least version `XXXXXX TBD XXXXXX` of `litellm` to be able to use it with `langfuse_otel`! 
```

Comment:
Sorry, I was ahead of muself. There seems to be `1.72.6rc`, so I thought `1.72.6` is going to have it in. But `1.72.6` has already been released (?) and it does not work in there... So I guess it's gonna be whatever minor or patch version is after `1.72.6`?